Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -697,6 +709,10 @@ getCityConfig cityConfig cityName = do
                             auto : {
                               freeSeconds : 3,
                               perMinCharges : 1.50
+                            },
```

Comment:
This change needs to be done on v1-configuration file as well

Reply:
The change in v1-configuration of JatriSaathiDriver is already done here: https://github.com/nammayatri/nammayatri/pull/7324/files#diff-b77f8c1ee31c0c828a30660ba2d89998d501c94e2fa079a2abd891fea856d4e7R389
Now, @Rohit4997 should we add in other merchant's config also without knowing the values `perMinCharges` for that  specific city of the merchant?

Reply:
Yes @slow-codex , It need's to be added in all other merchant's city config, Else configs will fail for other merchants
You can make `perMinCharges` 0 if you don't know the values, But better sync it with backend